examples/multi-chain-value/README.md

```diff
@@ -10,14 +10,15 @@ This example is currently deployed to [BSC Testnet](https://testnet.bscscan.com/
 
 ### Doing a cross-chain value transfer
 
-1. Get some [testnet Zeta](https://docs.zetachain.com/develop/get-testnet-zeta).
-1. Go to the [BSC Testnet Zeta token contract write methods](https://testnet.bscscan.com/address/0x6Cc37160976Bbd1AecB5Cce4C440B28e883c7898#writeContract).
-1. Connect with your wallet.
-1. Give allowance to the multi-chain value contract: `0x0E396e23cAD688c0e5252B20dFeAcC248b0d8B01`.
-1. Go to the [BSC Testnet contract's write methods](https://testnet.bscscan.com/address/0x0E396e23cAD688c0e5252B20dFeAcC248b0d8B01#writeContract).
-1. Connect with your wallet.
-1. Transfer an allowed amount of Zeta to Goerli (chain id 5). *Make sure to convert the destinationAddress to bytes, you can use [this code snippet](https://stackblitz.com/edit/typescript-bwhh4c?file=index.ts).*
-1. Check the receiver address balance in Goerli. *Note that the cross-chain transfer may take around 1 minute.*
+1. Get some [testnet Zeta](https://docs.zetachain.com/develop/get-testnet-zeta). You may have to manually add the ZETA token to your wallet. The token addresses can be found [here](https://docs.zetachain.com/develop/development-addresses).
+1. Go to the [BSC Testnet ZETA token contract's write methods](https://testnet.bscscan.com/address/0x6Cc37160976Bbd1AecB5Cce4C440B28e883c7898#writeContract).
+1. Connect your wallet.
+1. Give allowance to the multi-chain value contract to spend your ZETA: `0x0E396e23cAD688c0e5252B20dFeAcC248b0d8B01`.
+1. Go to the [BSC Testnet multi-chain value contract's write methods](https://testnet.bscscan.com/address/0x0E396e23cAD688c0e5252B20dFeAcC248b0d8B01#writeContract).
+1. Connect your wallet.
+2. You will need some tBNB to perform the transfer step. You can get some from [this faucet](https://testnet.binance.org/faucet-smart).
+3. Transfer an allowed amount of ZETA to Goerli (chain id 5) by calling `send()` in the write contract, passing in your wallet's address as the `destinationAddress`. *Make sure to convert the destinationAddress to bytes. You can use [this code snippet](https://stackblitz.com/edit/typescript-bwhh4c?file=index.ts).* For converting amounts, use https://eth-converter.com/ to express the amount in wei.
```

Comment:
Maybe we can change the copy from `Transfer an allowed amount of` to `Transfer (up to) the amount you previously approved` so it's more clear